packages/react-components/react-motion/stories/src/Tokens/Cards.tsx

```diff
@@ -0,0 +1,266 @@
+
+    display: 'flex',
+    flexDirection: 'column',
+    gap: '5px',
+
+    fontFamily: tokens.fontFamilyMonospace,
+    fontSize: tokens.fontSizeBase400,
+    padding: '5px 10px',
+    backgroundColor: tokens.colorNeutralBackground3,
+    borderRadius: tokens.borderRadiusMedium,
+  },
+  name: {
+    textAlign: 'center',
+  },
+  value: {
+    fontSize: tokens.fontSizeBase200,
+    fontStyle: 'italic',
+    whiteSpace: 'nowrap',
+    textAlign: 'center',
+  },
+});
+
+function getPointsFromEasing(value: string) {
+  if (value.startsWith('cubic-bezier')) {
+    const points = value.slice(13, -1).split(',');
+
+    if (points.length !== 4) {
+      throw new Error('Invalid cubic-bezier() value');
+    }
+
+    return points.map(v => parseFloat(v) * 100) as [number, number, number, number];
+  }
+
+  throw new Error('Only cubic-bezier() easing is supported');
+}
+
+const MotionCurveCard: React.FC<{ animationEnabled: boolean; tokenName: string; tokenValue: string }> = props => {
+  const { animationEnabled, tokenName, tokenValue } = props;
+
+  const classes = useCardClasses();
+  const easingPoints = getPointsFromEasing(tokenValue);
+
+  return (
+    <div className={classes.container}>
+      <div className={classes.graph}>
+        <svg className={classes.svg} viewBox="0 0 100 100">
+          <path
+            d={`M 0 100 C ${easingPoints[0]} ${easingPoints[1]}, ${easingPoints[2]} ${easingPoints[3]}, 100 0`}
+            fill="none"
+            stroke={tokens.colorNeutralStrokeAccessible}
+            strokeWidth="2"
+          />
+        </svg>
+        <div className={classes.graphX}>X</div>
+        <div className={classes.graphT}>t</div>
+      </div>
+      <div
+        className={classes.point}
+        style={{ animationPlayState: animationEnabled ? 'running' : 'paused', animationTimingFunction: tokenValue }}
+      />
+
+      <div className={classes.title}>
+        <div className={classes.name}>{tokenName}</div>
+        <Divider />
+        <div className={classes.value}>{tokenValue}</div>
+      </div>
+    </div>
+  );
+};
+
+const MotionDurationCard: React.FC<{ animationEnabled: boolean; tokenName: string; tokenValue: string }> = props => {
+  const { animationEnabled, tokenName, tokenValue } = props;
+  const classes = useCardClasses();
+
+  return (
+    <div className={classes.container}>
+      <div className={classes.graph}>
+        <div
+          className={classes.duration}
+          style={{
+            animationDuration: `${tokenValue}ms`,
+            animationPlayState: animationEnabled ? 'running' : 'paused',
+          }}
+        ></div>
+      </div>
+      <div className={classes.title}>
+        <div className={classes.name}>{tokenName}</div>
+        <Divider />
+        <div className={classes.value}>{tokenValue}ms</div>
+      </div>
+    </div>
+  );
+};
+
+export const MotionCurves = () => {
+  const classes = useClasses();
+  const [animationEnabled, setAnimationEnabled] = React.useState(true);
```

Comment:
In the previous implementation, we intentionally started with animations disabled to avoid any motion sickness.
Are we ok to enable it by default now (or do we want to be fancy and check `prefers-reduced-motion` for the initial state?

Reply:
Fixed it, using `.matchMedia()` to detect an initial state